lib/screens/app/profile/profile.dart

```diff
@@ -214,7 +230,7 @@ class _ProfileState extends State<Profile> {
   void _editProfilePicBottomSheet(BuildContext context) {
     Map<String, IconData> _items = {
       'Choose from Gallery': Icons.image,
-      'Click a picture': Icons.camera_alt,
+      'Take a picture': Icons.camera_alt,
```

Comment:
Don't know if it's important but Click a picture sounded weird

Reply:
Agree